DigitalLearningSolutions.Web/Views/LearningMenu/Index.cshtml

```diff
@@ -15,3 +15,12 @@
     <p class="nhsuk-u-secondary-text-color">@Model.CourseContent.BannerText</p>
   </div>
 </div>
+
+<div class="nhsuk-back-link">
+  <a class="nhsuk-back-link__link" asp-action="Current" asp-controller="LearningPortal">
+    <svg class="nhsuk-icon nhsuk-icon__chevron-left" focusable='false' xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+      <path d="M13.41 12l5.3-5.29a1 1 0 1 0-1.42-1.42L12 10.59l-5.29-5.3a1 1 0 0 0-1.42 1.42l5.3 5.29-5.3 5.29a1 1 0 0 0 0 1.42 1 1 0 0 0 1.42 0l5.29-5.3 5.29 5.3a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42z"></path>
+    </svg>
+    Return to Courses

```

Comment:
"Return to your current activities" might be better 🙂 ("activities" rather than "courses" because they've changed to use that terminology on the front end I've noticed, probably because it includes self assessments in addition to courses 🙂 )

Reply:
If we are referring to them as activities now, would we want to change the main page to have the phrase "Launch Activity" instead of "Launch Course" in a different ticket/PR

Reply:
No because that button only appears on courses 🙂

Reply:
Self assessments say "Launch self assessment"